api/python/provisioner/api.py

```diff
@@ -296,6 +296,33 @@ def sw_update(targets=ALL_MINIONS, nowait=False):
     )
 
 
+def rollback_update(target_version, targets=ALL_MINIONS, nowait=False):
```

Comment:
Guess we should change this to
**sw_rollback**

Reply:
addressed